build.gradle

```
@@ -1,17 +1,17 @@
 plugins {
-    id 'fabric-loom' version '0.9.+'
-    id 'io.github.juuxel.loom-quiltflower' version '1.3.+'
+    id 'fabric-loom' version '0.10-SNAPSHOT'
```

```suggestion
    id 'fabric-loom' version '0.10.+'
```

Consider the use of `-SNAPSHOT` artifacts very obselete, Gradle 7 introduced the `+` syntax that will fetch the latest version.